modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/ps/PostgisGroupByVisitorOnlineTest.java

```diff
@@ -73,6 +80,29 @@ public void testAggregateOnFunction(boolean expectOptimized) throws IOException
     }
 
     public void testTimestampHistogramDate() throws Exception {
-        testTimestampHistogram("last_update_date");
+        testTimestampHistogram("last_update_date", "d");
```

Comment:
The test inherited from the base class should be left intact, as guarantee of backwards compability, a new test can be added to handle time unit support. Actually you can do it in the base class, this will trigger tests in other databases as well and check the in memory evalution is also consistent.

Reply:
Yep. I have already restored the standard test whilst fixing the 3 Errors below.